### ISSUE TITLE
Labelled training data used in pMTnet

### ISSUE THREAD

**Issue by madnessfish** [tianshilu/pMTnet#6]
Thank you for a great tool! I am still pretty new in this field.

I would like to learn more about the training process on pMTnet. I am not sure if I missed the training data in the repository. Could you please provide the training data used in pMTnet with positive and negative labels (e.g. positive/TCR_output.csv, negative/TCR_output.csv, training_positive.csv)? Thank you so much for all your efforts!

**Comment by tianshilu** [comment]
Hi, 

Thanks for your interest. You can find positive training and negative training through the links below. https://drive.google.com/file/d/1_pf6xIK2dRql_zZ5A1BzoGvWIYVvEaBp/view?usp=sharing https://drive.google.com/file/d/1KLlH-CBS4ep6UAEeh4Zv9Eghk1hZUWj7/view?usp=sharing

Thanks!

**Comment by Miles-DDD** [comment]
Hi @tianshilu 
I have a  similar request. I'd like to make some comparisons on the method you proposed. Could you provide the testing data used in pMTnet with both pos/neg labels? Thank you!

**Comment by tianshilu** [comment]
Hi @Miles-DDD,

Please find the testing data with labels through the links below:
https://drive.google.com/file/d/1iddT16YEbEh5LYULokEMoey53RPiVsXt/view?usp=sharing
https://github.com/tianshilu/pMTnet/blob/master/test/input/test_input.csv

Thanks!

**Comment by madnessfish** [comment]
Hi @tianshilu 
Thank you for providing all the information!

I am curious about how the negative sets are generated (like any script?), as I have found 1912 entries are overlapping in the positive and negative training sets as the following command. Not sure if I have made any mistakes here.
`comm -12 <(sort -u neg_training.csv ) <(sort -u pos_training.csv ) | wc -l`

Also, I would like to know how these labeled training/ test data contribute to the training_data.csv and testing_data.csv under the pMTNet/data repository.
